docs/config/meta.md

```diff
@@ -9,6 +9,41 @@ E.g., registering a new tool configuration class in the `meta.tool_configs`
 section allows use of that class when configuring a custom tool in a given
 room.
 
+## Registering AG-UI Feature Classes
+
+The `meta.agui_features` section registers AG-UI feature types so that
+they can be referenced by their `name`.  Each feature is a contract
+between the client application and the server, defining the schema for
+a named field in the AG-UI state, and which parties are expected to write
+to that field.
+
+The section contains a list of mappings, each of which include:
+
+- `name`, a string identifying the field in the AG-UI state.
+
+- `model_klass`, a Python "dotted name" which can be used to import the
+   model class which defines the field's schema.
```

Comment:
A python dotted name (a fully qualified python class name?) serves the backend well but not the frontend.